runtime/common/NoiseModel.h

```diff
@@ -106,6 +117,12 @@ class kraus_channel {
   }
 
 public:
+  /// @brief Noise type enumeration
+  noise_model_type noise_type = noise_model_type::unknown;
+
+  /// @brief Noise probability
+  real probability = 0.0;
```

Comment:
Do we envision a need for multiple noise parameters?

Reply:
> Do we envision a need for multiple noise parameters?

Yes, I suppose that could happen for some future noise models. For example, if we wanted to mimic the [PAULI_CHANNEL_1](https://github.com/quantumlib/Stim/blob/main/doc/gates.md#the-pauli_channel_1-instruction) noise option from Stim, then that would need multiple parameters. Do you think we should change `probability` to a vector of doubles?

Reply:
Yea I think that would be a good idea, maybe call it parameters

Reply:
Updated in aa7982853